lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -1073,7 +1073,7 @@ sub save_user {
     use LedgerSMB::PGDate;
 
     _init_db($request);
-    $request->{dbh}->{AutoCommit} = 0;
+    $request->{dbh}->{AutoCommit} = 0; #Enable transactions, must manage.
```

Comment:
Can you explain what you want to tell the programmer reading this comment? Especially the "must manage" part? Doesn't the caller roll back on error and commit on success? Then isn't the transaction already managed?

Reply:
We decided to remove autocommit. We thus have to manage ourselves the commit and rollbacks.

Reply:
Ok, but is this a special case? As in: apparently, the programmer nweds to be made aware of this, because this is an exceptional case. Can you provide some insight into what is exceptional here? As per the "inline comments" section of our coding guide lines https://ledgersmb.org/content/coding-guidelines